pandas/util/testing.py

```diff
@@ -2088,7 +2088,7 @@ def dec(f):
 # and conditionally raise on these exception types
 _network_error_classes = (IOError, httplib.HTTPException)
 
-if sys.version_info >= (3, 3):
+if PY3:
```

Comment:
Since 3.0 - 3.4 are no longer supported, seems fine to use `PY3`.  Could use `PY35` if being explicit would be preferred here (or maybe just leave as-is).

Reply:
I'm fine with this change.  We should be encouraging code that relies solely on the supported versions of Python and not to the C-esque thing of making it super-backwards-compatible 😄